### PR TITLE
Auto bump kubevirt-ci to the latest release

### PR DESCRIPTION
**Pull request by nunnatsa**
This will allow adding the 1.21 lane.

Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly
  
**Release note**:

```release-note
None
```

### CODE REVIEW

File: cluster/kubevirtci.sh [hunk 1]
```diff
@@ -12,18 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-'k8s-1.19'}
-
-KUBEVIRTCI_VERSION="8f48705333a7b9c4c91cf8a0b96b40bb54ef6c8d"
-export KUBEVIRTCI_TAG=2103111738-8f48705
```
Comment (tiraboschi) [inline cluster/kubevirtci.sh:18]: I think that kubevirtci needs this.

Comment (nunnatsa) [reply cluster/kubevirtci.sh:18]: `KUBEVIRTCI_VERSION` is not in use in kubevirtci. It was used here to checkout a specific commit. I replaced it with checking out the tag.

Comment (tiraboschi) [reply cluster/kubevirtci.sh:18]: right, so I think we should alsop drop hack/bump-kubevirtci.sh

**Review by tiraboschi — APPROVED** [review]

**Pull request merged**